tests/foreman/ui/test_rhai.py

```diff
@@ -0,0 +1,136 @@
+"""Tests for Red Hat Access Insights"""
+
+
+from nailgun import entities
+from fauxfactory import gen_string
+from robottelo.common import conf
+from robottelo.common import manifests
+from robottelo.ui.navigator import Navigator
+from robottelo.ui.session import Session
+from robottelo.test import UITestCase
+from robottelo.vm import VirtualMachine
+
+
+class RHAITestCase(UITestCase):
+    def test_rhai_client_setup(self):
+        """@Test: Check for client registration to
+        redhat-access-insights service.
+
+        @Feature: RHEL client registration to rhai
+
+        @Assert: Registered client should appear in the Systems
+        sub-menu of Red Hat Access Insights
+        """
```

Comment:
Nitpick: Please write this docstring as shown in https://github.com/SatelliteQE/robottelo/pull/2567#issuecomment-121968947